pkg/utrace/utrace.go

```diff
@@ -642,15 +750,18 @@ func (u *UTrace) TraceEventsHandler(Cpu int, data []byte, perfMap *manager.PerfM
 		return
 	}
 
+	// resolve binary
+	binary := u.TracedBinaries[evt.Cookie]
+
 	// create new stack trace
-	stackTrace = NewStackTrace(1)
+	stackTrace = NewStackTrace(1, binary)
```

Comment:
We will need to change the identification of a stack trace / uprobe. It is not linked to a binary.
I think

Stack traces are all linked to a PID, which has an executable binary as its root (though that can change). uprobes are linked to a binary (though can use a PID optionally)